kernel/kernel-impl/src/main/java/org/sakaiproject/tasks/impl/TaskServiceImpl.java

```diff
@@ -95,7 +98,10 @@ public void update(Observable o, Object arg) {
             if (event.getEvent().equals(SiteService.SECURE_UPDATE_SITE_MEMBERSHIP)) {
                 try {
                     Set<String> siteUsers = siteService.getSite(event.getContext()).getUsers();
```

Comment:
I know this is out of scope for your PR, just an FYI for anyone following. event.getContext() is *usually* null here. also it can get called thousands of times in quick succession during SIS updates ;)